skflow/preprocessing/categorical.py

```diff
@@ -59,6 +60,10 @@ def fit(self, X, unused_y=None):
         Returns:
             self
         """
+        if HAS_PANDAS:
```

Comment:
Could this part be re-factored into some logic like setup_train_data_feeder or something else?